roles/mysql-5.6/tasks/mysql-server.yml

```diff
@@ -30,6 +30,7 @@
     priv: "*.*:ALL,GRANT"
   with_items:
     - "{{ ansible_hostname }}"
+    - "{{ openmrs_mysql_app_host }}"
```

Comment:
`openmrs_mysql_app_host` will not always be available to this role (especially if the inventory files aren't shared between OpenSRP and OpenMRS). I'd suggest you create a variable, let's say `mysql_app_host`, that you set when calling the mysql-5.6 role from OpenSRP's and OpenMRS's main meta files.